addons/web/static/src/xml/ribbon.xml

```diff
@@ -2,8 +2,10 @@
 <template>
 
     <t t-name="web.ribbon">
-        <div class="ribbon ribbon-top-right">
-            <span t-att-class="widget.bgColor ? widget.bgColor : 'bg-success'"><t t-esc="widget.text"/></span>
+        <div class="o_ribbon">
```

Comment:
in stable, you can't modify the structure of a template

Reply:
@kebeclibre I don't remeber why I did xml structure change but I removed it and tested  state selection widgets and ribbon and everything was working fine